readthedocs/api/v2/proxied_urls.py

```diff
@@ -0,0 +1,16 @@
+"""
+Proxied API URLs.
+
+Served from the same domain docs are served,
+so they can make use of features that require to have access to their cookies.
+"""
+
+from django.conf.urls import include, url
+
+from .views.proxied import ProxiedFooterHTML
+
+api_footer_urls = [
+    url(r'footer_html/', ProxiedFooterHTML.as_view(), name='footer_html'),
```

Comment:
We should add search in here while we are updating it. Probably also the sustainability API for completeness.